lampod/src/actions/handler.rs

```diff
@@ -131,14 +135,23 @@ impl Handler for LampoHandler {
                 push_msat,
                 channel_type,
             } => {
-                Err(error::anyhow!("Request for open a channel received, unfortunatly we do not support this feature yet."))
+                log::info!("Recieved open channel request!");
+                let liquidity_manager = self.liquidity_manager.clone();
+                if let Some(_) = liquidity_manager {
+                    let _ = self.channel_manager.channeld().accept_inbound_channel_from_trusted_peer_0conf(&temporary_channel_id, &counterparty_node_id, 0);
+                }
+                Ok(())
             }
             ldk::events::Event::ChannelReady {
                 channel_id,
                 user_channel_id,
                 counterparty_node_id,
                 channel_type,
             } => {
+                let liquidity_manager = self.liquidity_manager.clone();
+                if let Some(ref liq_manager) = liquidity_manager {
+                    let _ = liquidity_manager.unwrap().channel_ready(user_channel_id, &channel_id, &counterparty_node_id);
```

Comment:
we should do this only with the LSP not with any nodes

Reply:
Yes, we are doing this when we have some liquidity config inside the `lampo.conf`